test/functional/fixtures/api/es-next/navigate-to-and-test-page/testcafe-fixtures/navigate-to-test.js

```diff
@@ -1,11 +1,41 @@
+import path from 'path';
+import { Selector, ClientFunction } from 'testcafe';
+
 fixture `NavigateTo`
     .page `http://localhost:3000/fixtures/api/es-next/navigate-to-and-test-page/pages/index.html`;
 
+const pageHeader  = Selector('h1');
+const getLocation = ClientFunction(() => window.location.toString());
+
 test('Navigate to another page', async t => {
     await t.navigateTo('navigation.html')
         .click('#button');
 });
 
+test('Navigate to relative file page', async t => {
+    await t
+        .navigateTo('../pages/relative.html')
+        .expect(pageHeader.innerText).eql('Relative file page');
+});
+
+test('Navigate to absolute file page', async t => {
```

Comment:
Also I guess it will fix https://github.com/DevExpress/testcafe/issues/965. Worth adding a test. Also add it to `closes` directive